### DISCUSSION TITLE
What does "webhook definition" do？

### DISCUSSION THREAD

**Discussion by mylidong** [elsa-workflows/elsa-core#2832] [Q&A]

**Comment by sfmskywalker** [comment]
Hi @mylidong 

A webhook definition contains information about a webhook you want to receive from **other systems**.

Normally, you might do this by adding an **HTTP Endpoint** activity to your workflow.
But using the Webhooks module, you can more easily define these webhooks so that you can add these definitions to your workflows directly.

Under the hood, this module uses an activity type provider that provides preconfigured **HTTP Endpoint** activities based on each webhook definition. 

Basically, webhook definitions are translated into HTTP Endpoint activities made available from the activity picker.

Future updates might include insights in what webhooks were invoked how often, etc,
